software/mistserver.yml

```diff
@@ -1,8 +1,8 @@
 name: MistServer
-website_url: https://mistserver.org/
-description: Streaming media server that works well in any streaming environment.
+website_url: https://mistserver.com/
```

Comment:
I think the original `.org` domain is actually a better resource for self-hosting than the .com website which does not have any kind of technical documentation/info.

Reply:
this makes sense tbh

Reply:
Still includes a description of what the app is. Id also stick with the `.org`

Reply:
Yeah that makes sense, we'll make sure to give the `.org` website a visual touch up as well sometime soon but at least it links to all developer resources